python/samples/getting_started/00-getting-started.ipynb

```diff
@@ -109,6 +109,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Remove all services so that this cell can be re-run without restarting the kernel\n",
+    "kernel.remove_all_services()\n",
```

Comment:
we can also leverage `add_service(..., overwrite=True)` instead of this, but this is fine as well!